src/MBC_UserImport_Source_Niche.php

```diff
@@ -254,209 +195,270 @@ public function setter($message)
    */
   public function process()
   {
-
-    $payload = $this->addCommonPayload($this->importUser);
-    $existing['log-type'] = 'user-import-niche';
-    $existing['source'] = $payload['source'];
-
-    // Add welcome email details to payload
-    $this->addWelcomeEmailSettings($this->importUser, $payload);
-
-    // Check for existing email account in MailChimp
-    $subscribed = $this->mbcUserImportToolbox->checkExistingEmail(
-      $this->importUser,
-      $existing
-    );
-    if (!$subscribed) {
-      $this->addEmailSubscriptionSettings($this->importUser, $payload);
+    // Shortcuts.
+    $northstar = &$this->northstar;
+    $input = &$this->user;
```

Comment:
This feels like it's unnecessarily obscuring things for the sake of saving a few keystrokes.